tests/test_observation.py

```diff
@@ -18,3 +22,19 @@ def test_min_flux():
 
 def test_dm_delay():
     assert 1 == 1
+
+
+def test_dm_to_dist():
+    obs = Observation()
+
+    assert obs.dm_to_dist("00h00m00s", "00d00m00s", 0, "ne2001") == 0
+    assert obs.dm_to_dist("00h00m00s", "00d00m00s", 0, "ymw16") == 0
+
+    assert (
+        pytest.approx(obs.dm_to_dist("17h45m40s", "-29d00m28s", 10, "ne2001"), rel=1e-6)
+        == "422.4824905395508 pc"
+    )
```

Comment:
Not sure if this test would work, as "422.4824905395508 pc" is a string. I think it should be: 422.4824905395508

Reply:
@KshitijAggarwal You're right; I changed the function before pushing and forget to change the test accordingly.